company-profiles/wemakemvp.md

```diff
@@ -0,0 +1,35 @@
+# WeMakeMVP
+
+## Company blurb
+
+### About us
+
+We build web/mobile apps using the no-code platform bubble.io. 
+
+### Our mission
+
+Empower entrepreneurs by implementing their ideas despite the technical barriers. 
+
+## Company size
+
+The core team consists of 5 specialists.
+
+## Remote status
+
+Fully-remote
+
+## Region
+
+Ukraine (Worldwide)
```

Comment:
Can you clarify this part, please?

Reply:
Clarified this, added that founder is in Ukraine but the company accepts coworkers from all the globe.